src/Main.js

```diff
@@ -56,21 +57,22 @@ class Main extends React.Component {
          singleMovie: data.movie
        })
      })
-     .catch(error => console.log("error"))
+     .catch(error => this.setState({error: error}))
   }
 
   render() {
     console.log("render")
+    if(this.state.error) {
+      return (
+        <Error errorMessage={this.state.error.message}/>
+      )
+    }
     if (this.state.loading) {
-      console.log("movie data", this.state.movieData)
-      console.log("loading status", this.state.loading)
       return (
         <p>'Loading...'</p>
```

Comment:
Possibly expand on this styling - would it make sense to have a Loading component?